### PR TITLE
Feature/calendar grid

### PR DESCRIPTION
**Pull request by bonusrk**

### CODE REVIEW

File: src/app/core/state/calendar/calendar.state.ts [hunk 1]
```diff
@@ -51,6 +52,14 @@ export class CalendarState {
     });
   }
 
+  @Action(ToggleCalendarMode)
+  private toggleCalendarModel(ctx: StateContext<CalendarStateModel>): void {
```
Comment (vorant94) [inline src/app/core/state/calendar/calendar.state.ts:56]: typo toggleCalendarModel -> toggleCalendarMode

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes, { useHash: true })],
+  imports: [RouterModule.forRoot(routes, { useHash: false })],
```
Comment (vorant94) [inline src/app/app-routing.module.ts:26]: need to update nginx conf accordingly

File: src/app/calendar/calendar-header/calendar-header.component.html [hunk 3]
```diff
@@ -1,21 +1,31 @@
...
+    size="s"
+    type="button"
+    (click)="onCalendarNavigateButtonClicked('add')"
+    [icon]="'tuiIconChevronRightLarge'"
+  ></button>
+  <tui-toggle
+    size="l"
+    (click)="onModeToggle()"
+  ></tui-toggle>
+  <span>{{ hebrewMode ? 'Hebrew':'Gregorian' }}</span>
```
Comment (vorant94) [inline src/app/calendar/calendar-header/calendar-header.component.html:21]: maybe async pipe? (then there is no need to manage subs in ts)

**Review by vorant94 — CHANGES_REQUESTED** [review]

**Review by Feviad — APPROVED** [review]

**Comment by vorant94** [comment]
это по идее можно деклаинить с учетом перехода на реакт, так?

**Pull request closed without merging**